js/views/modals/Settings/Page.js

```diff
@@ -21,6 +22,11 @@ export default class extends baseVw {
 
     this.profile = app.profile.clone();
     this.listenTo(this.profile, 'sync', () => app.profile.set(this.profile.toJSON()));
+
+    this.socialAccounts = this.createChild(SocialAccounts, {
+      collection: this.profile.get('contactInfo').get('social').clone(),
```

Comment:
I'll take that out, it was to deal with the weird things happening because the ids didn't match, now that they do it's not needed.